spring-cloud-starter-task-spark-yarn/README.adoc

```diff
@@ -0,0 +1,35 @@
+//tag::ref-doc[]
+= Spark YARN Task
+
+This task is intended to launch a Spark application from a task. The task submits the Spark application to a YARN clusterfor execution. This task is appropriate for a deployment that has access to a Hadoop YARN cluster. The Spark application jar and the Spark Assembly jar should be referenced from an HDFS location. 
```

Comment:
First sentence should read: This task is intended to launch a Spark application. Will fix on merge